### PR TITLE
Rich content field starter

### PR DESCRIPTION
**Pull request by mcMyWave**

### CODE REVIEW

**Review by mywavecristian — APPROVED** [review]

File: examples/rich-content-field-react-starter/src/rcf_types.ts
Comment (reubenberghan) [inline examples/rich-content-field-react-starter/src/rcf_types.ts:1]: Why snake case this filename? I thought our pattern is to camel case our filenames.

**Review by Kai-mywave — APPROVED** [review]

File: examples/rich-content-field-react-starter/src/rcf_types.ts
Comment (mcMyWave) [reply examples/rich-content-field-react-starter/src/rcf_types.ts:1]: updated

Comment (reubenberghan) [reply examples/rich-content-field-react-starter/src/rcf_types.ts:1]: I don't see the updated file in the PR diff? 😅

**Comment by reubenberghan** [comment]
@mcMyWave - just an FYI the file `rcfTypes.ts` was not committed as part of this PR

**Comment by mcMyWave** [comment]
Thx @reubenberghan ! missed that

**Pull request merged**